alphadia/constants/default.yaml

```diff
@@ -11,6 +11,9 @@ general:
   wsl: false
   mmap_detector_events: false
   use_gpu: true
+  # whether to save the (MBR) library to the output directory
+  save_library: True
```

Comment:
I think we could add that the first lib is the input library and the second is the output library.